client/components/keyed-suggestions/index.jsx

```diff
@@ -330,28 +373,59 @@ class KeyedSuggestions extends Component {
 		} );
 	};
 
+	createTopLevelTermsSuggestions = () => {
+		const rendered = [
```

Comment:
[nit] I think we could return the `JSX.Element` directly and we don't need to follow the `createSuggestions`.

Reply:
Good idea, it looks cleaner!